README.md

```diff
@@ -1,5 +1,5 @@
 # MicroPython Rotary Encoder Driver
-A MicroPython driver to read a rotary encoder.  Works with Pyboard, ESP8266, and ESP32 development boards.  This is a robust implementation providing effective debouncing of encoder contacts.  It uses two GPIO pins configured to trigger interrupts, following Ben Buxton's implementation:
+A MicroPython driver to read a rotary encoder.  Works with Pyboard, Pico, ESP8266, and ESP32 development boards.  This is a robust implementation providing effective debouncing of encoder contacts.  It uses two GPIO pins configured to trigger interrupts, following Ben Buxton's implementation:
```

Comment:
It should likely be changed to `Raspberry Pi Pico` to be consistent with all the documentation, and to avoid confusion with the TinyPico (popular ESP32 board).

Reply:
Agreed; I will update the PR code accordingly.